packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -154,6 +183,7 @@ export class Environment extends EnvironmentBase {
       public readonly applicationId = applicationId;
       public readonly environmentId = environmentId;
       public readonly environmentArn = environmentArn;
+      public readonly name? = undefined;
```

Comment:
```suggestion
      public readonly name = undefined;
```

Reply:
I think having name as an optional property here is ok as it's only required when using the new methods.